week-4/C-more-complex-objects/exercise-3.js

```diff
@@ -32,16 +32,22 @@ var parkAvenueHouse = {
 
 // returns the full name (first name + last name) of the owner of the house
 function getOwnerFullName(house) {
+  return house.currentOwner.firstName.concat(house.currentOwner.lastName)

```

Comment:
What happens when you call this function with `kinningParkHouse`? Does it return the full name in the format you would expect?

Reply:
When I tried calling the function with KinningParkHouse, that is an error .... cannot ready properties of undefined, I have used the plus (+), operator instead